Mage.Server.Plugins/Mage.Player.Human/src/mage/player/human/HumanPlayer.java

```diff
@@ -2084,14 +2084,16 @@ public void selectBlockers(Ability source, Game game, UUID defendingPlayerId) {
         if (skipButtonActivated && !canStopOnAny && !canStopOnZero) {
             return;
         }
+        java.util.List<UUID> possibleBlockers = game.getBattlefield().getActivePermanents(filter, playerId, game).stream()
+                .map(p -> p.getId())
+                .collect(Collectors.toList());
+        // Skip prompt to select blockers if player has none
+        if (possibleBlockers.isEmpty()) return;
 
         while (canRespond()) {
             prepareForResponse(game);
             if (!isExecutingMacro()) {
                 Map<String, Serializable> options = new HashMap<>();
-                java.util.List<UUID> possibleBlockers = game.getBattlefield().getActivePermanents(filter, playerId, game).stream()
```

Comment:
Must keep that code cause possible blockers list must be collected after each target selection here. There are possible triggers and other game changes on single blocker declare.